### ISSUE TITLE
envify not wiring in process env vars correctly

### ISSUE THREAD

**Issue by cdaringe** [choojs/bankai#431]
🐛 bug report

## Expected Behavior

- setting `NODE_ENV=development npm start` should yield choo components with `process.env.NODE_ENV` popluated

## Current Behavior

- `NODE_ENV` is not reflected on the page when referenced
- here's a single commit on top of the default create-choo-app deployment, using bankai 9.7.0: https://github.com/cdaringe/choo-envify-/commit/7bf61d79204a726c7f51b353d9653264dc5f7f94

![untitled mov](https://user-images.githubusercontent.com/1003261/36403520-28442214-159a-11e8-89a9-9e8ac0aa0059.gif)

^you will notice that `development` flickers, then disappears.

## Possible Solution

- maybe SSR is getting the env, but the bundled app is dropping the env? that could explain the flickering.

## Code Sample

- see repo above

## Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Package          | create-choo-app/bankai|
| Runtime          | node latest |
| Package Manager  | npm |
| Operating System | osx, latest|

**Comment by goto-bus-stop** [comment]
Oh, this is because we use envify via tinyify, which is only enabled in `bankai build`. we should add it in an `else` here: https://github.com/choojs/bankai/blob/f0cb935f4d03eb42f2eb797f6de37ed070207c81/lib/graph-script.js#L100

**Comment by cdaringe** [comment]
i queued it up:

```js
  if (shouldMinify) {
    b.plugin(tinyify)
    b.on('split.pipeline', function (pipeline) {
      tinyify.applyToPipeline(pipeline, b._options)
    })
  } else {
    b.plugin(envify) // i required it in, above
  }
```

and npm linked it into create choo app.  the issue still occurs.  I confirmed that the envify code block is being hit as well.  any other ideas?

**Comment by goto-bus-stop** [comment]
envify is a transform, not a plugin:

```js
b.transform(envify, { global: true })
```

Altho I can't get that to work just yet either :sweat_smile:

**Comment by goto-bus-stop** [comment]
Should be fixed by #436 in :package: 9.8.0!

**Comment by cdaringe** [comment]
:tada:
